packages/node_modules/@node-red/nodes/core/io/21-httprequest.js

```diff
@@ -33,6 +33,7 @@ module.exports = function(RED) {
             var tlsNode = RED.nodes.getNode(n.tls);
         }
         this.ret = n.ret || "txt";
+        this.authType = n.authType;
```

Comment:
Normally we usually force the default here (as per line 35 above) - again making the following logic then just need to handle the new paths

    this.authType = n,authType || "basic";

Reply:
Hey Dave (@dceejay),
I had it like you say in my first version, but I rewrote it since I had to know (further in the code) whether it was undefined (i.e. version < 0.20.x).  But in the current implementation that isn't required anymore.  So I will change and test it this evening ...

Reply:
thanks - look forward to merging it tomorrow !